azure-pipelines.yml

```diff
@@ -94,6 +94,63 @@ stages:
 
   #######################################
 
+  - job: RPIBuilds
+    pool:
+      name: Default
```

Comment:
Is it ok to let the agent connected to "Default" pool?

I'm mean, if there are other jobs that are using "Default Pool" for building things, but without specifying an explicit agent, the build may go on this.